tools/shared/src/random-id.ts

```diff
@@ -0,0 +1,19 @@
+/*
+Copyright 2024 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+// This gnarly-looking implementation returns the equivalent of crypto.randomUUID().slice(0, 8).
+// It uses getRandomValues() in order to be compatible with HTTP contexts.
+export function randomID(): string {
```

Comment:
Thanks for weaving this into a helper method, I'd never know how to write this myself!

Reply:
This is one of the cases where the tests _actually_ came first for me, since this took a few tries to get right!

Reply:
Also once this is in & part of the performance tests, I'd like to make this more readable by breaking out its components into named assignments in series. That _may_ have no impact on its speed, but we won't know without measuring.